modules/apps/foundation/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/liferay/init.es.js

```diff
@@ -37,7 +37,7 @@ var initSPA = function(callback) {
 					}
 
 					var excluded = Liferay.SPA.excludedPaths.some(
-						(excludedPath) => url.indexOf(excludedPath) === 0
+						(excludedPath) => url.indexOf(themeDisplay.getPathContext() + excludedPath) === 0
```

Comment:
Hi @dacousalr, could you please help check @brunobasto mentioned concerns?
https://github.com/Robert-Frampton/liferay-portal/pull/381#discussion_r114856806

Regards,
Hai